src/ensembl/src/content/home/Home.scss

```diff
@@ -1,44 +1,223 @@
 @import 'src/styles/common';
 
 .home {
-  padding-bottom: 80px;
+  display: grid;
+  grid-template-rows: minmax(550px, auto) 85px;
+}
 
-  h2 {
-    color: $grey;
-    font-size: 14px;
-    font-weight: $normal;
-  }
+.container {
```

Comment:
I see the `.container` class describes the area of the home page that is above the footer. Maybe call it `.main`?

![image](https://user-images.githubusercontent.com/6834224/123648773-2323c800-d821-11eb-85ff-76707a529c0e.png)

Reply:
Done